### ISSUE TITLE
Temperature Converter

### ISSUE THREAD

**Issue by RahulScripted** [Evgenii-Bazhaov/Hacktoberfest2024#21]
I want to add a new project that is a temperature converter using HTML, CSS & JS. Here is the preview of the website - 
![Screenshot 2024-10-14 102712](https://github.com/user-attachments/assets/24a1cacd-69de-4b3d-ad79-209d4f987ddf)

Could you assign me under hacktoberfest, gssoc-ext and relative levels for this? Thank you @Evgenii-Bazhaov

**Comment by aryanayush012** [comment]
I want to work on it
please assign me this

**Comment by Evgenii-Bazhaov** [comment]
@RahulScripted  are u  working

**Comment by RahulScripted** [comment]
> @RahulScripted are u working

Yes, @Evgenii-Bazhaov  but it shows conflict because sir you accept someone else code after assigning it to me I've already built this can add it any time but it shows conflicts because of that reason.
